src/prototypes/entity_icons.lua

```diff
@@ -1,9 +1,8 @@
--- Auto generated by post.py, do not edit
```

Comment:
Is this file no longer generated by post.py? I do not see any changes to that file.

Reply:
Yeah, blender render is gone, I should remove it.